ui/components/app/terms-of-use-popup/terms-of-use-popup.test.js

```diff
@@ -20,16 +28,6 @@ const render = () => {
 };
 
 describe('TermsOfUsePopup', () => {
-  beforeEach(() => {
-    const mockIntersectionObserver = jest.fn();
```

Comment:
why was this here? were we using IntersectionObserver before and someone removed it in favor of `debounce`?! 🤔